### PR TITLE
feat: add select all logic for applyto in security policy

### PR DESCRIPTION
**Pull request by wu-cl**

### CODE REVIEW

File: pkg/agent/controller/policy/policy_controller.go [hunk 1]
```diff
@@ -306,6 +306,11 @@ func (r *Reconciler) completePolicy(policy *securityv1alpha1.SecurityPolicy) ([]
 		return nil, err
 	}
 
+	// if apply to is nil or empty, add all ips
+	if len(policy.Spec.AppliedTo) == 0 {
```
Comment (KeyongSun) [inline pkg/agent/controller/policy/policy_controller.go:310]: Do we need to check nil before get length?

File: pkg/webhook/validates/crd_validate_test.go [hunk 2]
```diff
@@ -353,9 +353,9 @@ var _ = Describe("CRD Validate", func() {
 				policy = securityPolicyIngress.DeepCopy()
 			})
 
-			It("Create policy with nil applied to should not allowed", func() {
+			It("Create policy with nil applied to should allowed", func() {
 				policy.Spec.AppliedTo = nil
-				Expect(validate.Validate(fakeAdmissionReview(policy, nil, "")).Allowed).Should(BeFalse())
+				Expect(validate.Validate(fakeAdmissionReview(policy, nil, "")).Allowed).Should(BeTrue())
```
Comment (KeyongSun) [inline pkg/webhook/validates/crd_validate_test.go:358]: policy.Spec.AppliedTo = nil or policy.Spec.AppliedTo = "" ?

File: pkg/agent/controller/policy/policy_controller.go (hunk 1 above)
Comment (wu-cl) [reply pkg/agent/controller/policy/policy_controller.go:310]: len(nil) will return 0

File: pkg/webhook/validates/crd_validate_test.go (hunk 2 above)
Comment (wu-cl) [reply pkg/webhook/validates/crd_validate_test.go:358]: nil or empty will transfer into same value.

File: pkg/agent/controller/policy/policy_controller.go (hunk 1 above)
Comment (KeyongSun) [reply pkg/agent/controller/policy/policy_controller.go:310]: OK

**Review by KeyongSun — APPROVED** [review]

**Pull request merged**